qemu/tests/x86_cpu_test_0x40000001.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from virttest import error_context
+from virttest import data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check kvm 0x40000001 inside guest.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    nums_cpu = params.get("smp")
```

Comment:
Can not be `0` so `params['smp']` better?

Reply:
will get smp by `vm.cpuinfo.smp`, thanks.